### PR TITLE
changes steady receipe

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
About The Pull Request

changes steady from needing carpotoxin to needing roach chems 

## Why It's Good For The Game

Needing an RNG drop that you probably won't get for chemistry is lame. Half the time, the ship does not move. There is generally no carp on 60% of rounds. Sad!

## Changelog
```changelog
tweak: steady receipe change
```

### CODE REVIEW

File: code/modules/reagents/recipes.dm [hunk 1]
```diff
@@ -688,7 +688,7 @@
 	holder.del_reagent("napalm")
 	return
 
-/datum/chemical_reaction/chemsmoke
+
```
Comment (Michiyamenotehifunana) [inline code/modules/reagents/recipes.dm:691]: wtf is this

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
It appears that a line of code was accidentally deleted. Please try to restore it.

File: code/modules/reagents/recipes.dm (hunk 1 above)
Comment (bearrrrrrrr) [reply code/modules/reagents/recipes.dm:691]: what the ufck-

**Comment by bearrrrrrrr** [comment]
fixed
not sure wtf happened there lmao i guess i accidentally deleted it somehow

**Review by Michiyamenotehifunana — APPROVED** [review]
All good!

**Pull request merged**